src/main/java/uk/gov/justice/digital/service/JDBCGlueConnectionDetailsService.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.justice.digital.service;
+
+
+import jakarta.inject.Inject;
+import jakarta.inject.Singleton;
+import lombok.val;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.gov.justice.digital.client.glue.GlueClient;
+import uk.gov.justice.digital.client.secretsmanager.SecretsManagerClient;
+import uk.gov.justice.digital.config.JobArguments;
+import uk.gov.justice.digital.datahub.model.JDBCCredentials;
+import uk.gov.justice.digital.datahub.model.JDBCGlueConnectionDetails;
+
+import java.util.Map;
+
+/**
+ * Responsible for retrieving JDBC details from AWS Glue Connections.
+ */
+@Singleton
+public class JDBCGlueConnectionDetailsService {
```

Comment:
This is actually a class rename with a minor change as this was previously specific to the operational datastore.